bot/controllers/newUsers.js

```diff
@@ -0,0 +1,18 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+
+class newUserController extends BaseController {
+  constructor(message) {
+    super(message);
+    const controller = this;
+    this.commands = [
+      new Command('!welcome', '!welcome', 'Welcome Message', 'Sends a new user of the channel the welcome message', this.msgNewUserWelcome.bind(controller), 'dm'),
```

Comment:
Your implementation is deviating from in the formatting from the other established controllers.